Quick copy updates to the README file (#8)
mui/tech-challenge-design-engineer: README.md

Head over to [our company Handbook](https://mui-org.notion.site/Why-MUI-d8b8c142a6a44e3aa963f26edf4e03db) to learn more!

## Challenge

Given that the Design Engineer position sits right at the intersection between engineering and design, this challenge is tailored to demonstrate your skills in both areas.
Our challenge is to **develop a stacked Snackbar** component.

Here's a reference design:

<img alt="Snackbars example design" src="./public/snackbar-reference.gif" width="465" height="392" />

Note that it's only an example.

> We encourage you to add your creative spin to the design; we want to see your design taste!

Reviewer: punctuation nitpick 😜

```suggestion
We encourage you to add your creative spin to the design—we want to see your design taste!
```

## Requirements

> - **Stack:** Use Base UI's [`UnstyledSnackbar`](https://mui.com/base/react-snackbar/) component or another headless component library as its foundation.

Reviewer: ```suggestion
- **Stack:** Use the Base UI [Snackbar](https://mui.com/base/react-snackbar/) or a similar headless component from a library of your choice as the foundation.
```

- **Interactivity:**

>   - The other Snackbar instances should be visible after some interaction, either clicking a button or hovering the stack.

Reviewer: ```suggestion
  - The other Snackbar instances should be visible after some interaction, either clicking a button or hovering over the stack.
```

>   - They should be kept open if the focus is inside them. Conversely, they should collapse as the focus moves out of it.

Reviewer: ```suggestion
  - They should stay open when focused, and collapse when the focus moves away.
```

>   - Closing them should be possible by clicking on a close button.

Reviewer: ```suggestion
  - Each instance should have a functional close button.
```

>   - _Bonus:_ adding an option to close it after some configurable time interval. It should still be compatible with [WCAG 2.2.1](https://www.w3.org/TR/WCAG21/#enough-time).

Reviewer: ```suggestion
  - _Bonus:_ Add an option to make the Snackbars close automatically after some configurable time interval. This should be compatible with [WCAG 2.2.1](https://www.w3.org/TR/WCAG21/#enough-time).
```

- **Transitions:** the Snackbars should appear and disappear gracefully on the screen, using an animation.

> - **Dark mode:** make sure to support light and dark modes.

Reviewer: These are not "bonuses" right? If so, maybe they should go above the one Bonus on the list for the sake of clarity?

## Work environment

> To save you time, we created a working environment with Next.js, Typescript, eslint, and others, that you should install.

Reviewer: ```suggestion
To save you time, we created a working environment with Next.js, Typescript, eslint, and other complementary dependencies.
```

Follow the steps:

- Clone the repo: `git clone git@github.com:mui/tech-challenge-design-engineer.git`
- Install the dependencies: `yarn`
- Start Next.js: `yarn dev`
- Open http://0.0.0.0:3002/

> ## Submission instructions

Reviewer 2: There is a potential to apply the same improvements in

- https://github.com/mui/tech-challenge-full-stack#submission
- https://github.com/mui/tech-challenge-react#submission
